src/auro-checkbox.js

```diff
@@ -50,6 +50,15 @@ class AuroCheckbox extends LitElement {
     };
   }
 
+  // This custom event is only for the purpose of supporting IE
+  // .addEventListener('change', function() { })
+  change(event) {
```

Comment:
```suggestion
  handleChange(event) {
```
For consistency with input handler